### ISSUE TITLE
Menu music respects in-game volume slider, yet cannot change it

### ISSUE THREAD

**Issue by Birdulon** [ZeroK-RTS/Zero-K#4130]
Apologies if there's another issue for this, I just did a quick search for "volume".

Using the top volume slider in-game to lower the volume seems to affect the main menu music volume as well, however there doesn't seem to be a way to change that from the lobby. It would be nice if something linked to the top volume slider also appeared in the main menu, e.g. to the left of the "Home" and "Logout/Login" buttons.

**Comment by Sharkgamedev** [comment]
I agree if there isn't one, although I had originally thought that there was an option in settings? I'll check in a bit to be sure.

**Comment by Sharkgamedev** [comment]
https://github.com/ZeroK-RTS/Chobby/pull/769

**Comment by GoogleFrog** [comment]
I don't think the ingame and lobby music should be sharing a setting, and a slider isn't a replacement for an option in the lobby settings menu.

**Comment by Birdulon** [comment]
Yeah, only the master volume is shared. In my implementation I add the lobby music slider underneath the master slider to mirror the master+in-game music sliders.

**Comment by Arch-Shaman** [comment]
Note: For future reference, menu (eg lobby) is under 'chobby' here. Zero-K only covers in-game stuff.
Otherwise, thanks for the ticket!
